terraform/aws/national-parks.tf

```diff
@@ -51,6 +51,7 @@ resource "aws_instance" "permanent_peer" {
       "chmod +x /tmp/install_hab.sh",
       "sudo /tmp/install_hab.sh",
       "sudo hab license accept",
+      "sudo hab pkg install core/hab-sup/0.83.0-dev -c unstable",
```

Comment:
Should we parametrize this so we can do something like,
`"sudo hab pkg install ${var.hab-sup-version}",`

That way we can have `core/hab-sup` as a default variable, and `core/hab-sup/0.83.0-dev -c unstable` in tfvars or similar

Reply:
I was thinking about that, but I also think that line is pretty temporary. 0.83.0 will be stable in a week or so and we can remove that line.

Reply:
True enough. 👍 
Might want to throw in a comment or something in the associated TF files, or a GH issue to track the removal so it doesn't become forgotten cruft.

Reply:
I went ahead and made the sup a variable, it'll help now and will be useful if we ever have a Hab sup upgrade that breaks the demo.